src/main/groovy/org/craftercms/cli/commands/site/PublishContent.groovy

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2007-2024 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.craftercms.cli.commands.site
+
+import org.craftercms.cli.commands.AbstractCommand
+import org.craftercms.cli.options.SiteOptions
+import picocli.CommandLine
+
+// Command to publish content
+@CommandLine.Command(name = 'publish-content', description = 'Publish content from a project or site.')
+class PublishContent extends AbstractCommand {
+
+    @CommandLine.Mixin
+    SiteOptions siteOptions
+
+    @CommandLine.Option(names = ['--items'], description = 'The items to publish')
+    String items
+
+    @CommandLine.Option(names = ['--optionalDependencies'], description = 'The optional dependencies')
```

Comment:
Optional dependencies is a boolean flag AFAIK. If on, optional deps are included in the package, else they're not.

Reply:
Agreed to support this in `4.1`, and then change it to be a boolean in `develop`